src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -243,6 +251,29 @@ def _agentpool_format_group(item):
     ])
 
 
+def _connected_acr_format_group(item):
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('STATUS', _get_value(item, 'status')),
+        ('PARENT', _get_value(item, 'parent')),
+        ('LOGIN NAME', _get_value(item, 'loginName')),
```

Comment:
@rosanch please validate that the dictionary entries here map to the correct property in the model. For instance, LOGIN NAME corresponds to `_get_value(item, 'loginServer', 'host')`